### PR TITLE
PRIME-2320 Oral Health Professional (re-work)

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs [hunk 1]
```diff
@@ -103,6 +103,23 @@ public override IEnumerable<License> SeedData
...
+                    new License { Code = 93, Weight = 7, Name = "Limited (Armed Forces/Government) Dentist" },
+                    new License { Code = 94, Weight = 8, Name = "Dental Therapist" },
+                    new License { Code = 95, Weight = 9, Name = "Dental Technician" },
+                    new License { Code = 96, Weight = 10, Name = "Full Denturist" },
+                    new License { Code = 97, Weight = 11, Name = "Student Dentistry" },
+                    new License { Code = 98, Weight = 12, Name = "Limited (Grandparented) Denturist" },
+                    new License { Code = 99, Weight = 13, Name = "Student dental technology" },
+                    new License { Code = 100, Weight = 14, Name = "Student Denturism" },
+                    new License { Code = 101, Weight = 15, Name = "Temporary" },
+
```
Comment (neophyte57) [inline prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs:122]: Please order as per the list in https://bcgovmoh.atlassian.net/browse/PRIME-2320?focusedCommentId=15098

**Comment by neophyte57** [comment]
I had setup a situation like this (everything working fine)

image


I selected "College of Oral Health Professionals", then "Full Dentist", got to Review screen and then selected Edit to come back to College Licence Information screen, where I saw this unexpectedly  

image

File: prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs (hunk 1 above)
Comment (bergomi02) [reply prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs]: total forgot about it. it is fixed.

**Comment by bergomi02** [comment]
> I had setup a situation like this (everything working fine)
> 
> image
> 
> I selected "College of Oral Health Professionals", then "Full Dentist", got to Review screen and then selected Edit to come back to College Licence Information screen, where I saw this unexpectedly
> 
> image

Good catch! There is bug and it is fixed now.

**Comment by bergomi02** [comment]
The implementation has been reworked to another other PR. closing this PR.

**Pull request closed without merging**